WordPress/gutenberg
# Add reset button to color control

## What?


Add an inline reset button to the color control, shown on hover/focus, if there's a value. Picked changes from [this PR](https://github.com/WordPress/gutenberg/pull/50989) with a few tweaks. The original PR was closed as stale, but I believe this is a nice improvement that will improve the overall experience for creators.

Props @kevin940726 

## Why?


To make it easier to reset a color selection, reduce the number of clicks as it doesn't require opening the color options panel.

Fixes https://github.com/WordPress/gutenberg/issues/41866 

## How?

Adding a button if there's a color value, and adjusting the CSS so that it looks like the proposed designs.

## Testing Instructions

1. Create a page.
2. Add a paragraph.
3. Add some colors to the paragraph (background/text)
4. Confirm that for the elements that have a color, if you hover/focus you have a "reset" option and that it resets the color selection.

## Screenshots or screencast 

https://github.com/user-attachments/assets/5e41d3ca-be1c-43ad-980c-c26b07515505
